# Requires the `data-test` attribute on <a> elements.

## Rule Details

Examples of **incorrect** code for this rule:

```html
<a href="example.com">Download</a>
```

Examples of **correct** code for this rule:

```html
<a href="example.com" data-test="download-link">Download</a>
```

