# Requires the `data-test` attribute on <input> elements.

## Rule Details

Examples of **incorrect** code for this rule:

```html
<input type="text" />
```

Examples of **correct** code for this rule:

```html
<input type="text" data-test="name" />
<input type="text" readonly />
```

